application/src/main/java/org/togetherjava/tjbot/commands/tag/TagManageCommand.java

```diff
@@ -127,7 +120,7 @@ private void editIdTag(SlashCommandEvent event) {
                     && ex.getErrorResponse() == ErrorResponse.UNKNOWN_MESSAGE) {
                 event.reply("This message doesn't exist").setEphemeral(true).queue();
             } else {
-                throw new RuntimeException(failure);
+                failure.printStackTrace();
```

Comment:
Please dont print the stack trace. Either rethrow or use the logging system. `log.error(failure)` or something.
I would prefer a rethrow though, then we could react on it and handle it higher up in the logic - if needed.

Reply:
> I would prefer a rethrow though

Rethrowing is not possible here, since it's a `Consumer`.

Reply:
Done by printing the exception to the logger